app/src/main/java/com/myapps/toualbiamine/basketbusiness/Model/User.java

```diff
@@ -0,0 +1,38 @@
+package com.myapps.toualbiamine.basketbusiness.Model;
+
+
+public class User {
+        private String name;
+        private String password;
+        private String email;
+        private boolean isStaff;
+
+    public User() {
+
+}
+
+    public User(String email, String name, String password, Boolean isStaff) {
+        this.email = email;
+        this.name = name;
+        this.password = password;
+        this.isStaff = isStaff;
+}
+
+    public String getEmail() { return email; }
+
+    public void setEmail(String email) { this.email = email; }
+
+    public String getName() { return name; }
+
+    public void setName(String name) { this.name = name; }
+
+    public String getPassword() { return password; }
+
+    public void setPassword(String Password) { this.password = password; }
```

Comment:
Please, make it `setPassword(String password)`. We don't give capital letter to the first word in variable or function!

Reply:
So you understand, the issue will be that when you say `this.password = password`, we are basically saying 'The password of THIS user is going to be set to the parameter of the function named password'. 
The problem is that the parameter is called `Password` which won't be recognized correctly.